zsh/.zshrc

```diff
@@ -102,14 +102,17 @@ export LSCOLORS='Exfxcxdxbxegedabagacad'
 export LS_COLORS='di=1;34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
 
 load-our-ssh-keys() {
-  if [ -z "$SSH_AUTH_SOCK" ]; then
-   # Check for a currently running instance of the agent
-   RUNNING_AGENT="$(ps -ax | grep 'ssh-agent -s' | grep -v grep | wc -l | tr -d '[:space:]')"
-   if [ "$RUNNING_AGENT" = "0" ]; then
-        # Launch a new instance of the agent
-        ssh-agent -s &> ~/.ssh/ssh-agent
-   fi
-   eval $(cat ~/.ssh/ssh-agent)
+  if can_haz op; then export SSH_AUTH_SOCK=~/Library/Group\ Containers/2BUA8C4S2C.com.1password/t/agent.sock
```

Comment:
`/Library/Group\ Containers/` does't exist on my Ventura machine. I'm using `op` and have my `SSH_AUTH_SOCK` set to `/private/tmp/com.apple.launchd.VYdKrAMX96/Listeners`. Did you have to create it while installing `op`?

Reply:
Hi Joe, this folder should come with the install of 1Password 8. Here is more info [from the docs.](https://developer.1password.com/docs/ssh/get-started)

Reply:
I'll take a look at the docs again today and let you know what I think. If it turns out that it generates a different ugly prefix to `com.1password` per machine it is probably best to go with adding a "After you install" section in the docs with a link to 1p's site.

Reply:
I *think* everything is resolved and clean for your consideration.